framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/Deployment.java

```diff
@@ -17,10 +17,39 @@
 
 import java.util.List;
 
+/**
+ * Deployment representation in cloud environment.
+ */
 public interface Deployment {
+
+    /**
+     * Return deployment name. E.g. for Openshift project name.
```

Comment:
Namespace isn't a deployment name. It is an identifier grouping (one or several) deployments into logical groups.